frontend/src/components/Calendar.tsx

```diff
@@ -107,7 +104,11 @@ function Calendar({
   };
 
   const handleEventRemove = async (removeInfo: EventRemoveArg) => {
-    await removeEventFn(removeInfo);
+    const { event } = removeInfo;
+
+    if (isTimeInPast(event.start || new Date())) {
```

Comment:
Luenko ihan väärin, vai palauttaako isTimeInPast false jos aika on menneisyydessä ja true jos tulevaisuudessa? Jos näin on, niin nimeämistä voisi selkeyttää.